packages/node_modules/samples/browser-socket/app.js

```diff
@@ -66,20 +66,45 @@ document.getElementById('credentials').addEventListener('submit', (event) => {
 
   authorize()
     .then(() => {
+      console.log('connected');
       webex.messages.listen()
         .then(() => {
-          console.log('connected');
-          console.log('listening to messages');
+          console.log('listening to message events');
           updateStatus(true);
           webex.messages.on('created', (message) => {
-            console.log('created');
+            console.log('message created event:');
             console.log(message);
           });
         })
         .catch((err) => {
           console.error(`error listening to messages: ${err}`);
           updateStatus(false);
         });
+      webex.memberships.listen()
+        .then(() => {
+          console.log('listening to membership events');
+          updateStatus(true);
+          webex.memberships.on('created', (membership) => {
```

Comment:
Would be awesome to use the `SDK_EVENT` constants here.

Reply:
Good idea.  Updated.